ps.go

```diff
@@ -0,0 +1,76 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/codegangsta/cli"
+)
+
+var psCommand = cli.Command{
+	Name:      "ps",
+	Usage:     "ps displays the running processes of a container",
```

Comment:
Maybe reword this a bit "ps displays the processes running inside a container" ?